locallib.php

```diff
@@ -195,6 +203,12 @@ function local_bulkenrol_parse_emails($emails) {
     }
 }
 
+/**
+ * Takes an e-mail and returns a moodle user record and error string (if occured).
+ *
+ * @param string $email E-mail used to search for a user
+ * @return multitype:NULL |multitype:string mixed
```

Comment:
This type declaration seems really wrong. See https://docs.phpdoc.org/guides/types.html#multiple-types-combined for details